NEWS.md

```diff
@@ -1,3 +1,5 @@
-# admiralpeds 0.1.0.9000
+# admiralpeds 0.1.0.9001
```

Comment:
@kabis-ops no need to update NEWS file (sorry i know it says about this in the PR checklist) - for the very first release of a package though we just need the one liner i had there. as users only need to follow a changelog for subsequent releases

Reply:
> @kabis-ops no need to update NEWS file (sorry i know it says about this in the PR checklist) - for the very first release of a package though we just need the one liner i had there. as users only need to follow a changelog for subsequent releases

Ah yes of course, thanks Ross!